docs/README.md

```diff
@@ -1,4 +1,4 @@
 README
 ======
 
-This file is a placeholder. This directory will contain public documentation.
+This file is a placeholder.  This directory will contain public documentation.
```

Comment:
BUT I LOOK AT THE SOURCE IN VI AND TWO SPACES ARE WRONG.

Pro tip: If you're losing an argument, use all caps. This helps convincing people.